linux/net/rina/dtp-ps-default.c

```diff
@@ -84,6 +84,7 @@ int
 default_closed_window(struct dtp_ps * ps, struct pdu * pdu)
```

Comment:
That was a rage comment about decrements an unsigned int without checking for overflows (well, it's a valid condition for every variable at the end) . :scream:

I can remove it now, no prob.

About that additional if condition: it was there only for debugging purposes, so I'm removing it right now.